README.md

```diff
@@ -23,7 +23,12 @@ Note: modify the configs in basic_config.py before run the test script
 2. modify server/toolchain_git_branch_commit
 3. modify auth/https/serice ports if need..
 '''
-python deploy_start.py all # or 'server' or 'toolchain'
+python deploy_start.py all # or 'server' or 'toolchain' or 'hugegraph'
```

Comment:
hugegraph represents "server + pd + store"?

Reply:
yes, use hugegraph to start a cluster including 1 pd, 1 store and 1 server